webviz_subsurface/plugins/_simulation_time_series/views/subplot_view/_settings/_ensembles.py

```diff
@@ -0,0 +1,221 @@
+                        EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLES_STORE
+                    ),
+                    data=[],
+                ),  # TODO: Add predefined deltas?
+            ]
+        )
+
+    def __delta_ensemble_table_layout(self) -> dash_table.DataTable:
+        return dash_table.DataTable(
+            id=self.register_component_unique_id(
+                EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLE_NAMES_TABLE
+            ),
+            columns=(
+                [
+                    {
+                        "id": self.register_component_unique_id(
+                            EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLE_NAMES_TABLE_COLUMN
+                        ),
+                        "name": "Created Delta (A-B)",
+                    }
+                ]
+            ),
+            data=[],
+            fixed_rows={"headers": True},
+            style_as_list_view=True,
+            style_cell={"textAlign": "left"},
+            style_header={"fontWeight": "bold"},
+            style_table={
+                "maxHeight": "150px",
+                "overflowY": "auto",
+            },
+            editable=False,
+        )
+
+    def set_callbacks(self) -> None:
+        @callback(
+            [
+                Output(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLES_STORE
+                    ).to_string(),
+                    "data",
+                ),
+                Output(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLE_NAMES_TABLE
+                    ).to_string(),
+                    "data",
+                ),
+                Output(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.ENSEMBLES_DROPDOWN
+                    ).to_string(),
+                    "options",
+                ),
+            ],
+            [
+                Input(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.DELTA_ENSEMBLE_CREATE_BUTTON
+                    ).to_string(),
+                    "n_clicks",
+                )
+            ],
+            [
+                State(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLES_STORE
+                    ).to_string(),
+                    "data",
+                ),
+                State(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.DELTA_ENSEMBLE_A_DROPDOWN
+                    ).to_string(),
+                    "value",
+                ),
+                State(
+                    self.component_unique_id(
+                        EnsemblesSettings.Ids.DELTA_ENSEMBLE_B_DROPDOWN
+                    ).to_string(),
+                    "value",
+                ),
+            ],
```

Comment:
According to the Dash source code, providing `DashDependencies` in a list is considered the "old" syntax. 

```python
# flatten args, to support the older syntax where outputs, inputs, and states
# each needed to be in their own list
```

Consider providing the dependencies directly.